src/inspector_agent.cc

```diff
@@ -532,16 +591,13 @@ Agent::Agent(Environment* env) : parent_env_(env),
 
 // Destructor needs to be defined here in implementation file as the header
 // does not have full definition of some classes.
-Agent::~Agent() {
-}
+Agent::~Agent() = default;
 
 bool Agent::Start(node::NodePlatform* platform, const char* path,
                   const DebugOptions& options) {
   path_ = path == nullptr ? "" : path;
   debug_options_ = options;
-  client_ =
-      std::shared_ptr<NodeInspectorClient>(
-          new NodeInspectorClient(parent_env_, platform));
+  client_ = std::make_shared<NodeInspectorClient>(parent_env_, platform);
```

Comment:
My understanding is that make_shared should be fine as it is a part of C++11 (make_unique was introduced shortly after C++11)